src/laplace.jl

```diff
@@ -1,3 +1,4 @@
+using BlockDiagonals
 using .Curvature
 using Flux
 using Flux.Optimise: Adam, update!
```

Comment:
inconsistent style, are you using `using ...` or `using ...: ...`?

Reply:
We are using both